packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -276,6 +286,8 @@ export class TokenDetectionController extends StaticIntervalPollingController<To
     trackMetaMetricsEvent,
     messenger,
     useAccountsAPI = true,
+    useTokenDetection = true,
+    useExternalServices = true,
```

Comment:
I'm not sure if `useTokenDetection` can be toggled on/off at runtime. But for `externalServices` ("basic functionality"), you can toggle this on/off at runtime, which could make this field stale.

Replace with a function to remove the chance of it being stale: `() => boolean`